test/common/markdown/package-lock.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "ansi-colors": {
       "version": "1.1.0",
-      "resolved": "http://registry.npmjs.org/ansi-colors/-/ansi-colors-1.1.0.tgz",
+      "resolved": "https://registry.npmjs.org/ansi-colors/-/ansi-colors-1.1.0.tgz",
```

Comment:
Do we really need the file package-lock.json?

Reply:
I think its better to secure links to `npm` to prevent package intercepts